tests/utils/PolygonUtilsTest.cpp

```diff
@@ -353,19 +367,19 @@ struct GetNextParallelIntersectionParameters
     bool forward;
     coord_t dist;
 
-    GetNextParallelIntersectionParameters(const std::optional<Point> predicted, const Point start_point, const Point line_to, const bool forward, const coord_t dist)
-    : predicted(predicted)
-    , start_point(start_point)
-    , line_to(line_to)
-    , forward(forward)
-    , dist(dist)
+    GetNextParallelIntersectionParameters(const std::optional<Point> predicted,
+                                          const Point start_point,
+                                          const Point line_to,
+                                          const bool forward,
+                                          const coord_t dist)
```

Comment:
These lines used to be formatted with the comma in front, such that adding or removing a parameter doesn't cause different lines to change than the line of the parameter itself. Why did you remove that?